deploy/eck-stack/charts/eck-kibana/examples/http-configuration.yaml

```diff
@@ -13,25 +13,24 @@ labels: {}
 annotations: {}
   # key: value
 
-spec:
```

Comment:
should we add a comment saying this spec. is deprecated and config. is the preferred way to set these configs. Not sure what the right place is to add this comment

Reply:
Thanks, LGTM.